VERSION_MANAGERS.md

```diff
@@ -1,3 +1,7 @@
+## asdf
+
+If you use `asdf` and the VS Code extension fails to activate the environment, try running `asdf update` and restart VS Code.
```

Comment:
How about we mention the type of error that we've received reports for (the JSON parse error)?

Reply:
I thought we plan to improve the message as well?

Reply:
We do, but it felt weird to me to say that if anything fails with ASDF try to update it. I would include a link to one of the reported issues at least so that people can dig more into it and figure out if they are facing the same scenario.